README.md

```diff
@@ -62,190 +51,63 @@ The new registry implementation provides the following benefits:
-
-```
-go get github.com/tools/godep github.com/golang/lint/golint
-```
-
-**TODO(stevvooe):** Add a `make setup` command to Makefile to run this. Have
-to think about how to interact with Godeps properly.
-
-Once these commands are available in the `GOPATH`, run `make` to get a full
-build:
-
-```
-$ GOPATH=`godep path`:$GOPATH make
-+ clean
-+ fmt
-+ vet
-+ lint
-+ build
-github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar
-github.com/Sirupsen/logrus
-github.com/docker/libtrust
-...
-github.com/yvasiyarov/gorelic
-github.com/docker/distribution/registry/handlers
-github.com/docker/distribution/cmd/registry
-+ test
-...
-ok    github.com/docker/distribution/digest 7.875s
-ok    github.com/docker/distribution/manifest 0.028s
-ok    github.com/docker/distribution/notifications  17.322s
-?     github.com/docker/distribution/registry [no test files]
-ok    github.com/docker/distribution/registry/api/v2  0.101s
-?     github.com/docker/distribution/registry/auth  [no test files]
-ok    github.com/docker/distribution/registry/auth/silly  0.011s
-...
-+ /Users/sday/go/src/github.com/docker/distribution/bin/registry
-+ /Users/sday/go/src/github.com/docker/distribution/bin/registry-api-descriptor-template
-+ /Users/sday/go/src/github.com/docker/distribution/bin/dist
-+ binaries
-```
-
-The above provides a repeatable build using the contents of the vendored
-Godeps directory. This includes formatting, vetting, linting, building,
-testing and generating tagged binaries. We can verify this worked by running
-the registry binary generated in the "./bin" directory:
-
-```sh
-$ ./bin/registry -version
-./bin/registry github.com/docker/distribution v2.0.0-alpha.2-80-g16d8b2c.m
-```
-
-### Developing
-
-The above approaches are helpful for small experimentation. If more complex
-tasks are at hand, it is recommended to employ the full power of `godep`.
-
-The Makefile is designed to have its `GOPATH` defined externally. This allows
-one to experiment with various development environment setups. This is
-primarily useful when testing upstream bugfixes, by modifying local code. This
-can be demonstrated using `godep` to migrate the `GOPATH` to use the specified
-dependencies. The `GOPATH` can be migrated to the current package versions
-declared in `Godeps` with the following command:
-
-```sh
-godep restore
-```
-
-> **WARNING:** This command will checkout versions of the code specified in
-> Godeps/Godeps.json, modifying the contents of `GOPATH`. If this is
-> undesired, it is recommended to create a workspace devoted to work on the
-> _Distribution_ project.
-
-With a successful run of the above command, one can now use `make` without
-specifying the `GOPATH`:
-
-```sh
-$ make
-```
-
-If that is successful, standard `go` commands, such as `go test` should work,
-per package, without issue.
-
-Support
--------
+The majority of Docker users pull images from the Docker public registry
```

Comment:
This statement still doesn't belong here. It represents an opinion or a desired state. Really, we need to answer the question of why one would want to deploy their own registry. We need to be agnostic in our recommended use of the hub.

We should mention that this version is the same version running on the hub.